tools/istio-clean-iptables/pkg/cmd/cleanup.go

```diff
@@ -69,7 +69,12 @@ func removeOldChains(cfg *config.Config, ext dep.Dependencies, cmd string) {
 	chains := []string{constants.ISTIOOUTPUT, constants.ISTIOINBOUND}
 	flushAndDeleteChains(ext, cmd, constants.NAT, chains)
 	// Flush and delete the istio chains from MANGLE table.
-	chains = []string{constants.ISTIOINBOUND, constants.ISTIODIVERT, constants.ISTIOTPROXY}
+	if cfg.InboundInterceptionMode == constants.TPROXY {
+		chains = []string{constants.ISTIOINBOUND, constants.ISTIODIVERT, constants.ISTIOTPROXY, constants.OUTPUT, constants.PREROUTING}
+	} else {
+		chains = []string{constants.ISTIOINBOUND, constants.ISTIODIVERT, constants.ISTIOTPROXY}
+	}
+
 	flushAndDeleteChains(ext, cmd, constants.MANGLE, chains)
```

Comment:
is this going to flush the entire OUTPUT chain?

Reply:
yes. when  InboundInterceptionMode is TPROXY will build iptables rule
···
* mangle
-A PREROUTING -p tcp -m mark --mark 1337 -j CONNMARK --save-mark
-A OUTPUT -p tcp -m connmark --mark 1337 -j CONNMARK --restore-mark
···

Reply:
But users may have non-istio rules in OUTPUT

Reply:
yes you are right , i will update pull request

Reply:
@howardjohn   i updated , please code review。